tasmota/support_features.ino

```diff
@@ -554,7 +554,9 @@ void GetFeatures(void)
 #ifdef USE_PING
   feature6 |= 0x00000080;  // xdrv_38_ping.ino
 #endif
-
+#ifdef USE_THERMOSTAT
+   feature6 |= 0x00000200; // xsns_68_opentherm.ino
```

Comment:
0x00000100 is reserved for the https://github.com/arendst/Tasmota/pull/8212

Reply:
Fine